types/tiny-templater/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for tiny-templater 1.0
+// Project: https://github.com/jarofghosts/tiny-templater#readme
+// Definitions by: Jimmy Cuadra <https://github.com/jimmycuadra>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = template;
+
+type Data = string | { [key: string]: Data };
```

Comment:
I don't think it's valid to pass in a string to the template function. Maybe something like this works instead?

```suggestion
interface Data { [key: string]: DataElement };
type DataElement = string | Data;
```